src/main/java/cs361/battleships/models/Game.java

```diff
@@ -24,7 +29,7 @@ public boolean placeShip(Ship ship, int x, char y, boolean isVertical) {
         do {
             // AI places random ships, so it might try and place overlapping ships
             // let it try until it gets it right
-            opponentPlacedSuccessfully = opponentsBoard.placeShip(ship, randRow(), randCol(), randVertical());
+            opponentPlacedSuccessfully = opponentsBoard.placeShip(ship, randCol(), randVertical());
```

Comment:
Why did you delete randRow()? Or am I reading this wrong? The placeShip method requires four parameters, it looks like there are only three. This would not compile, right?